server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -481,8 +481,14 @@ export function configureWebSocketServices(
             };
         }
 
+        // server side listening for pong events.
+        socket.on("pong", (cb: any) => {
```

Comment:
This is a duplicate handler to the one added in services-shared/src/socketIoServer.ts